examples/evaluator-rules/src/main/resources/example.rules.kts

```diff
@@ -264,6 +264,28 @@ fun RuleSet.deprecatedScopeExcludeReasonInOrtYmlRule() = ortResultRule("DEPRECAT
     }
 }
 
+fun RuleSet.missingCiConfigurationRule() = projectSourceRule("MISSING_CI_CONFIGURATION") {
+    require {
+        -AnyOf(
+            projectSourceHasFile(
+                ".appveyor.yml",
+                ".bitbucket-pipelines.yml",
+                ".gitlab-ci.yml",
+                ".travis.yml"
+            ),
+            projectSourceHasDirectory(
+                ".circleci",
+                ".github"
+            )
+        )
+    }
+
+    error(
+        message = "This project does not seem to have any known CI configuration files.",
+        howToFix = "Please setup a CI. If you already have setup a CI and the error persists, please contact support."
```

Comment:
Mentioning support is a bit weird in an example rules file, better suggest to check if the CI file is contained in the file patterns.

Reply:
If someone has CI setup, then the rule reports a false positive. 
In that case I believe that the devs responsible for the rules have to look into it anyway.
So, IMO it does not make sense to ask the user to look into any further details of the heuristic first.

Can we keep it, also for simplicity and for illustrating how the process could work?

Reply:
This implies a context that the reader of this file is likely unaware of, but ok.